scripts/superdesk-desks/desks.js

```diff
@@ -172,7 +172,7 @@
                 }
 
                 scope.$watch('filter', queryItems);
-                scope.$on('task:stage', function(event, data) {
+                scope.$on('task:stage', function($event, data) {
```

Comment:
The event listeners that you see using `$event` are using Angular's [`$event`](https://docs.angularjs.org/guide/expression#-event-), which is fine. In this case this is a custom event so `e` or `event` should be better to avoid confusing the two concepts. Just like above where you've used `e` yourself.